### PR TITLE
Metadata editor fixes

### PR DESCRIPTION
**Pull request by harrywhite4**

### CODE REVIEW

File: python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/advanced_editor.html [hunk 1]
```diff
@@ -1,101 +1,82 @@
...
-            {% if request.is_ajax %}
-                <button accesskey="c" type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
-            {% else %}
-                <a accesskey="c" class="btn btn-default" href="{% url 'aristotle:item' item.id %}">Cancel</a>
-            {% endif %}
-            <input accesskey="s" type="submit" class="btn btn-primary" value="Save changes" />
-        </div>
+          <div class="form-footer">
+              {# Submit buttons #}
+              {% if request.is_ajax %}
```
Comment (s-i-l-k-e) [inline python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/advanced_editor.html]: let's get rid of this ajax

Comment (LegoStormtroopr) [reply python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/advanced_editor.html]: We aren't ajaxing the editor anymore are we?

Comment (s-i-l-k-e) [reply python/aristotle-metadata-registry/aristotle_mdr/templates/aristotle_mdr/actions/advanced_editor.html]: Yeah, this has become a bit of a leftover

**Comment by s-i-l-k-e** [comment]
Approving and merging because of an absence of changes that would affect display.

**Pull request merged**